src/main/java/org/primefaces/component/organigram/OrganigramHelper.java

```diff
@@ -96,8 +96,8 @@ public static OrganigramNode findTreeNode(OrganigramNode searchRoot, OrganigramN
      * @param node The root node.
      */
     public static void buildRowKeys(OrganigramNode node) {
-        if (node.getChildren() != null && !node.getChildren().isEmpty()) {
-            for (int i = 0; i < node.getChildren().size(); i++) {
+        if (node.getChildren() != null && node.getChildCount() > 0) {
```

Comment:
switch those statements

Reply:
actually do i need to check for NULL children at all of childCount>0 they can't be NULL right?  I think I can just remove the NULL check.

Reply:
unsure :D i wouldnt remove it, just to be sure